sdk/mapper.js

```diff
@@ -66,8 +66,12 @@ class Mapper {
         }
       }
     };
-    if (responseApi.hasReturn()) {
-      payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
+    try {
+      if (responseApi.hasReturn()) {
+        payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
```

Comment:
Actually, getReturn() is not implemented for the Response class, also not covered by tests either there or in Action class.